docs-source/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -102,6 +106,8 @@ The following updates are *supported* except when they reference an area that is
 
 ##### Unsupported updates
 
+TBD Add information about online updates...
```

Comment:
Marking the TBD so that we don't forget about it.

Reply:
Done.